graphstorm-processing/graphstorm_processing/data_transformations/dist_feature_transformer.py

```diff
@@ -38,14 +39,20 @@ class DistFeatureTransformer(object):
     """
 
     def __init__(
-        self, feature_config: FeatureConfig, spark: SparkSession, json_representation: dict
+        self,
+        feature_config: FeatureConfig,
+        spark: SparkSession,
+        json_representation: dict,
+        edge_mapping_dict: dict = None,
```

Comment:
I'd prefer if we didn't change the function's signature to support a single transformation if possible? 

Is it possible to provide this information inside the `feature_config.transformation_kwargs`? We can modify its value adding this dict as an additional key, only in the case where we get `feature_config.feat_type == "edge_dst_hard_negative"`